src/shared/selectors/flags.ts

```diff
@@ -91,6 +92,7 @@ export const CLOUD_FLAGS = {
   boardWithFlows: false,
   createWithFlows: false,
   leadWithFlows: false,
+  selfRemovalFromAccount: false,
```

Comment:
without getting into too much detail and history of what this file is, you don't need to add this here since `selfRemovalFromAccount` is a flag in ConfigCat. This file is more historical and was used before we had a legitimate feature flagging service. Flags here will only be settable via the user's localStorage, so they're machine dependent and can't be changed in the ConfigCat UI.